### ISSUE TITLE
Not able to read GV config from consul server for the application image built using installers

### ISSUE THREAD

**Issue by jgirase** [TIBCOSoftware/be-tools#20]
Steps:
1.Build the application image using installers (docker/bin) folder with some GV
2.Run Consul server 
3.Create GV/key-value pairs in consul server same as define in BE project
4.Run application image using below command,
docker run -d -e CONSUL_SERVER_URL=http://consul:8500 -e APP_CONFIG_PROFILE=default -e BE_APP_NAME=TestGV --net=mybridge --name=testgv testgv:560installer
5.Application does not read GV values from consul server.It also shows below error message in logs,
INFO: Reading GV values from Consul..
INFO: CONSUL_SERVER_URL = http://consul:8500
INFO: BE_APP_NAME = TestGV
INFO: APP_CONFIG_PROFILE = default
INFO: Reading GV values from Consul.. (TestGV/default/)
./gvproviders/consul/run.sh: line 28: /home/tibco/be/gvproviders/consul/consul: No such file or directory
./gvproviders/consul/run.sh: line 28: /home/tibco/be/gvproviders/consul/jq: No such file or directory

Basically it failed to download consul cli inside application image.
